src/reanimated2/animation/spring.ts

```diff
@@ -49,6 +50,7 @@ export const withSpring = ((
       duration: 2000,
       dampingRatio: 0.5,
       reduceMotion: undefined,
+      clamp: undefined,
```

Comment:
Can we simply avoid defining it here?

Reply:
No, we can't do it without changing our types